src/main/scala/subsystem/BaseSubsystem.scala

```diff
@@ -33,8 +31,11 @@ abstract class BareSubsystemModuleImp[+L <: BareSubsystem](_outer: L) extends La
   ElaborationArtefacts.add("dts", outer.dts)
   ElaborationArtefacts.add("json", outer.json)
   ElaborationArtefacts.add("plusArgs", PlusArgArtefacts.serialize_cHeader)
-  ElaborationArtefacts.add("objectModel.json", outer.objectModelJson)
   println(outer.dts)
+
+  def addOM(): Unit = {
```

Comment:
Can you rename this to `addOMArtefact()` so that it's clearer that this adds an ElaborationArtefact?